media/stylus/components/home/home-masthead.styl

```diff
@@ -25,11 +29,17 @@ homepage masthead (search, feature callouts)
         big-search();
     }
 
+    .search-icon {
+    		position: absolute;
+    		top: ($grid-spacing * 2 + 7);
+    }
+
     input#home-q {
         set-placeholder-style(letter-spacing, -1px);
         set-placeholder-style(color, $text-color);
         color: $text-color;
         background-color: #fff;
+        padding-left: 30px;
```

Comment:
We have wiki article in languages that go right to left so we have to code for text going in that direction as well. There's a stylus mixin to simplify that: https://github.com/mozilla/kuma/blob/58dae9a95be2f5fc1bfe4ed377baff0fbdc51c2a/media/stylus/includes/mixins.styl#L316-L346 an example usage would be:
`bidi-style(padding-left, 30px, padding-right, 0)`

And you can test by using the dev tools to change the `<html dir="ltr">` to `rtl`